.github/workflows/test.yml

```diff
@@ -68,6 +68,8 @@ jobs:
           sudo mkdir -p "${{ matrix.compose_path }}"
           sudo curl -L https://github.com/docker/compose/releases/download/${{ matrix.compose_version }}/docker-compose-`uname -s`-`uname -m` -o "${{ matrix.compose_path }}/docker-compose"
           sudo chmod +x "${{ matrix.compose_path }}/docker-compose"
+      - name: Install sentry-cli
```

Comment:
This adds a dependency on sentry-cli, I expect that is ok to add as a CI-only dependency, will add it to the e2e tests action if we go ahead with that. I couldn't use the docker container because on macos the network runs in a VM which is a different network than the host :/